scripts/qa_crowbarsetup.sh

```diff
@@ -442,9 +442,9 @@ function export_tftpboot_repos_dir()
     if iscloudver 5plus; then
         tftpboot_suse12_dir=/srv/tftpboot/suse-12.0
 
```

Comment:
It needs to change as it's used to reference the directory that contains `install/`. So I guess we need a new code path specific to 6plus...